### PR TITLE
fix(DB/DBC): correct gtoctclasscombatratingscalar_dbc

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Changes Proposed:
- Replace ID's to ID+1

## Issues Addressed:
- https://wow.tools/dbc/?dbc=gtoctclasscombatratingscalar&build=3.3.5.12340#page=1

## Tests Performed:
-

## How to Test the Changes:
- TODO

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1609109042026647800.sql [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+CREATE TABLE `gtoctclasscombatratingscalar_dbc`  (
+  `ID` int(11) NOT NULL DEFAULT 0,
+  `Data` float NOT NULL DEFAULT 0,
+  PRIMARY KEY (`ID`) USING BTREE
+) ENGINE = MyISAM CHARACTER SET = utf8 COLLATE = utf8_general_ci ROW_FORMAT = Fixed;
+
+-- ----------------------------
+-- Records of gtoctclasscombatratingscalar_dbc
+-- ----------------------------
+INSERT INTO `gtoctclasscombatratingscalar_dbc` VALUES (1, 1);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1609109042026647800.sql:16]: why so many `INSERT INTO` when you can just use one as explained here: https://www.azerothcore.org/wiki/Dealing-with-SQL-files#write-compact-code

Comment (Winfidonarleyan) [reply data/sql/updates/pending_db_world/rev_1609109042026647800.sql:16]: okay

File: data/sql/updates/pending_db_world/rev_1609109042026647800.sql [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+-- ----------------------------
+-- Table structure for gtoctclasscombatratingscalar_dbc
+-- ----------------------------
+DROP TABLE IF EXISTS `gtoctclasscombatratingscalar_dbc`;
+CREATE TABLE `gtoctclasscombatratingscalar_dbc`  (
+  `ID` int(11) NOT NULL DEFAULT 0,
+  `Data` float NOT NULL DEFAULT 0,
+  PRIMARY KEY (`ID`) USING BTREE
+) ENGINE = MyISAM CHARACTER SET = utf8 COLLATE = utf8_general_ci ROW_FORMAT = Fixed;
+
```
Comment (BarbzYHOOL) [inline data/sql/updates/pending_db_world/rev_1609109042026647800.sql:12]: 1) wasn't the table already existing? why not truncate it instead?
@Winfidonarleyan @FrancescoBorzi 

2) it's the issue reported by Wotex?

**Comment by Yehonal** [comment]
Yes, yes and yes

Il ven 1 gen 2021, 17:54 Barbz <notifications@github.com> ha scritto:

**Comment by Footman** [comment]
reported by me and tested already for long time. can be merged

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**